mlfinlab/portfolio_optimization/risk_estimators.py

```diff
@@ -258,3 +260,197 @@ def denoise_covariance(self, cov, tn_relation, kde_bwidth):
         cov_denoised = self.corr_to_cov(corr, np.diag(cov) ** (1 / 2))
 
         return cov_denoised
+
+    @staticmethod
+    def minimum_covariance_determinant(returns, price_data=False, assume_centered=False,
+                                       support_fraction=None, random_state=None):
+        """
+        Calculates the Minimum Covariance Determinant for a dataframe of asset prices or returns.
```

Comment:
Add more detailed docstings that also explain the method and link to supporting documentation such as sklearn. Its ok to say that this function is a wrapper for sklean.

Reply:
Will do